code/modules/jobs/job_types/psychiatrist.dm

```diff
@@ -0,0 +1,55 @@
+/datum/job/psychiatrist
+	title = "Psychiatrist"
+	flag = PSYCHIATRIST
+	department_head = list("Head of Security")
```

Comment:
supposed to be Chief Medical Officer I assume

Reply:
oh SHIT ye. i copied the whole code from the detective one , and it seems i forgot about  changing that